src/GridList.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module GridList
+ */
+class GridList extends React.Component {
+	render() {
+		const {
+			className,
+			columns,
+			items,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'gridList',
+			{
+				[`gridList--has${columns.default}`]: !!columns.default,
+				[`atMedium_gridList--has${columns.medium}`]: !!columns.medium,
+				[`atLarge_gridList--has${columns.large}`]: !!columns.large
+			},
+			className
+		);
+
+		return (
+			<ul
+				className={classNames}
+				{...other}>
```

Comment:
super obnoxious nitpick on my part can you do `>` on new line makes it easier to read